operator/Makefile

```diff
@@ -169,7 +169,7 @@ release-image.arm64v8: clean
 	docker build --load --build-arg ARCH="arm64v8" --build-arg RELEASE_VERSION="$(RELEASE_VERSION)" -t $(RELEASE_REGISTRY)/$(RELEASE_IMAGE)-arm64 .
 
 .PHONY: push-release-images
-push-release-images: release-image.amd64 release-image.arm64v8
```

Comment:
any special reason why we don't depend on `release-image.amd64` anymore?

Reply:
we call the 2 things separately now so we don't need the dependency here anymore

Reply:
But as a developer, I would prefer calling a single make entry and get both executed

Reply:
nvm, we have `build-and-push-local:`